package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "style-guide",
+  "version": "0.0.0",
+  "author": "Incuna",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/incuna/style-guide"
+  },
+  "bugs": {
+    "url": "https://github.com/incuna/style-guide/issues"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "grunt": "~0.4.1",
+    "grunt-contrib-connect": "~0.5.0",
+    "grunt-contrib-cssmin": "~0.7.0",
+    "grunt-contrib-watch": "~0.5.3",
+    "grunt-sass": "1.1.0",
+    "grunt-swig-templates": "^0.1.4",
+    "grunt-version": "^0.3.0",
```

Comment:
Pint exact versions here. `npm list --depth 0` will give you the versions you have installed, which may not be what's here minus `^` or `~`